app/Policies/MacAddressPolicy.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Policies;
+
+use App\MacAddress;
+use App\User;
+use App\Role;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class MacAddressPolicy
+{
+    use HandlesAuthorization;
+
+    public function before($user, $ability)
+    {
+        if ($user->hasRole(Role::INTERNET_ADMIN)) {
+            return true;
+        }
+        if (!$user->hasRole(Role::INTERNET_USER)){
```

Comment:
Please use the auto-formatting capabilities of PhpStorm. There is a missing whitespace before the `{`.